shoop/testing/factories.py

```diff
@@ -393,7 +394,7 @@ def get_completed_order_status():
     return OrderStatus.objects.get_default_complete()
 
 
-def create_product(sku, shop=None, supplier=None):
+def create_product(sku, shop=None, supplier=None, default_price=None):
```

Comment:
If this is `default_price=0`, you can just put it in the `ShopProduct.objects.create()` call (as 0 is the default anyway)